integration_test/support/models.exs

```diff
@@ -109,11 +122,29 @@ defmodule Ecto.Integration.User do
   schema "users" do
     field :name, :string
     has_many :comments, Ecto.Integration.Comment, foreign_key: :author_id
+    has_many :posts, Ecto.Integration.Post, dependent: :nothing, foreign_key: :author_id
+    has_many :addresses, Ecto.Integration.Address, dependent: :nilify_all
     belongs_to :custom, Ecto.Integration.Custom, references: :bid, type: :binary_id
     timestamps
   end
 end
 
+defmodule Ecto.Integration.Address do
```

Comment:
Could we get by without adding a new model? Or It seems we could add dependent to has_many :comments in the table above.